src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -53,6 +62,17 @@ export class TestRunner extends NodejsEnvTestRunner {
     _getPreparePageActions(browser: Browser, history: BrowserHistory): (() => Promise<void>)[] {
         return [
             async (): Promise<void> => {
+                this._socket.on(BrowserEventNames.runBrowserCommand, this._handleRunBrowserCommand(browser));
+
+                this._socket.emit(WorkerEventNames.initialize, {
+                    file: this._file,
+                    sessionId: this._runOpts.sessionId,
+                    capabilities: this._runOpts.sessionCaps,
+                    requestedCapabilities: this._runOpts.sessionOpts.capabilities,
+                    customCommands: browser.customCommands,
+                    config: this._config as BrowserConfig,
```

Comment:
in addition to pass `file`, I now need to transfer data to properly create the browser on client